src/MapsWithSearch/Input/index.js

```diff
@@ -73,13 +73,13 @@ class LocationSearchInput extends React.Component {
 
   handleSelect = address => {
     this.textInput.current.blur();
-    recordSearchCompleted();
     geocodeByAddress(address)
       .then(async results => {
         this.setState({ address });
         const result = results[0];
         const latLng = await getLatLng(result);
         console.log("Success", latLng);
+        recordSearchCompleted(result.types);
```

Comment:
I thought we want to see what are users searching for not what the api returns. @asanjeevrao is this what you meant